Core/PrimaryGenerator.cpp

```diff
@@ -65,10 +65,10 @@ G4PrimaryVertex* PrimaryGenerator::GenerateThreeGammaVertex(
   Double_t weight_max = event.GetWtMax() * pow(10, -1);
   Double_t rwt = 1;
   Double_t M_max = 1;
-  if(channel == MaterialExtension::DecayChannel::Ortho3G || channel == MaterialExtension::DecayChannel::Direct) {
+  if (channel == DecayChannel::Para3G) {
+    M_max = 2.00967 * pow(10, 25);
```

Comment:
I think it would be useful to add a comment why such values are used: `2.00967 * pow(10, 25)` one could also define some more meaningful const double variables with some name.

Reply:
I will add the reference to manual and the new version of the manual in the next commit